problems/arrayMethods.js

```diff
@@ -91,15 +143,31 @@ const targetCount = () => {};
  * @returns {Object} A mapping of elements to their occurrence count.
  */
 
-const counterObject = () => {};
+const counterObject = (items) => {
+  let object = {};
+  items.map((el) => {
```

Comment:
Map is not the right array method for this. You are using this for side effects so it should be a forEach. Map is only for returning a new array where every element has been acted on by the callback.